### ISSUE TITLE
python should be available

### ISSUE THREAD

**Issue by JasonGross** [coq-community/docker-coq#28]
`python` is not available in the images and `sudo apt-get install python` fails in the images, see https://github.com/coq-community/run-coq-bug-minimizer/runs/2114884688?check_suite_focus=true#step:4:234

It should probably be installed by default, since python (python3, in particular) is needed for the pretty-timing targets in coq-makefile.

**Comment by JasonGross** [comment]
See also https://github.com/coq-community/docker-coq-action/issues/53

**Comment by erikmd** [comment]
Thanks for opening this issue. The asnwer is twofold:

* `python` had not been manually added to the `coqorg/base` images as in Debian releases, python=python2; it will soon been unmaintained/deprecated if not already, but `python3` is already available by default if your script is compatible:
```
$ docker run --rm -it coqorg/coq:8.13 python3 --version
Python 3.7.3
```

* at first sight [the failure](https://github.com/coq-community/run-coq-bug-minimizer/runs/2114884688?check_suite_focus=true) you got looks normal, and it is documented here:
  https://github.com/coq-community/docker-coq/wiki/CI-setup#remarks  
  
  to sum up: the APT cache has been removed on purpose in all images, to earn some MBs regarding the size of the image), so you'd need to replace [this line](https://github.com/coq-community/run-coq-bug-minimizer/blob/9929ce9706286d0045baa88208de52bcdc17a889/run.sh#L35) with:

```
sudo apt-get update -y -q
sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -q python
```
or
```
which python || { sudo apt-get update -y -q; sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -q python; }
```

hoping this helps :)

**Comment by JasonGross** [comment]
Ah, okay, thanks
